packages/odyssey-react/src/components/Icon/Icon.module.scss

```diff
@@ -12,7 +12,7 @@
 
 .root {
   position: relative;
-  inset-block-start: -0.1em;
+  inset-block-start: -0.0625em;
```

Comment:
🤔

Reply:
the -0.1em was rendered out as a fractional pixel value, which lead to inconsistencies. The new value translates to a whole pixel value at the default text size.